packages/eslint-plugin/src/configs/eslint-recommended-raw.ts

```diff
@@ -7,7 +7,7 @@
  * - disables rules from eslint:recommended which are already handled by TypeScript.
  * - enables rules that make sense due to TS's typechecking / transpilation.
  */
-export default (
```

Comment:
Why did you switch this from an `export default`?

Reply:
Basically the same answer as [this comment](https://github.com/typescript-eslint/typescript-eslint/pull/10049#discussion_r1775961226), with a `default` export we get:

![tseslint-eslint-recommended-raw-with-default-export](https://github.com/user-attachments/assets/710561ab-1b77-4ca0-bd55-c16dbfbd0237)